components/shared/feature-flagged.ts

```diff
@@ -0,0 +1,14 @@
+import { FeatureFlag } from "lib/utils/server/feature-flags";
+
+interface FeatureFlaggedProps {
+  flag: FeatureFlag;
+  children: React.ReactElement;
+  featureFlags: Record<FeatureFlag, boolean>;
+}
+export const FeatureFlagged = ({ flag, featureFlags, children }: FeatureFlaggedProps) => {
```

Comment:
A wrapper component to render a component or not that works server and client side.

Reply:
This could also be done with e.g.

`{ featureFlags["contributions_evolution_by_type"] && <Component /> }`

but it feels more natural to use JSX in JSXland.

Reply:
I defer to engineers, but this is good to know.

Reply:
I guess it's a preference as well, so if folks don't like it, we can do the alternate way I mentioned. Not a hill I want to die on. 😅

Reply:
I don't have a strong opinion on it, but I agree with using JSX in JSXland 😅

Reply:
This works for me!